RELEASE.md

```diff
@@ -5,6 +5,13 @@ Please follow the established format:
 - Use present tense (e.g. 'Add new feature')
 - Include the ID number for the related PR (or PRs) in parentheses
 -->
+# Next release
+
+## Major features and improvements
+
+## Bug fixes and other changes
+- Upgrade the gitpod workspace-full to a newer version which includes both Node 18 and Python 3.9. (#1862)
```

Comment:
Is it using 3.9 or 3.12?

Reply:
ah yeah sorry it's 3.12, thanks spotting it.

Reply:
at this moment viz actively supports python 3.9-3.11 . It might work for 3.12 but we do not have it in our build. So either we can choose a different gitpod image or go with 3.12

Reply:
@ravi-kumar-pilla That's always an option, though you could just select a tag a bit earlier that would gives you Python 3.11 instead of Python 3.12.

For full flexibility, you can always go with custom image, the tradeoff is a slower build since we don't have the pre-build options with open source plan.

Reply:
yup agree !

Reply:
may be publish our custom image to dockerhub and use it :)

Reply:
hey I'm okay with either way, but i'd push to use the existing image for now because i think if it works fine and we don't need extra docker file to make it work, then why not? 

If we run to the similar issue again, in which we find ourself needing to find a specific version to fix our need, then maybe that will be the right time to customise it? 

what do you think? @ravi-kumar-pilla @noklam @rashidakanchwala @astrojuanlu